plugins/services/src/js/structs/Framework.js

```diff
@@ -47,7 +49,7 @@ module.exports = class Framework extends Application {
       }
       if (task.statuses != null) {
         return task.statuses.reduce(function(memo, status) {
-          if (status.healthy) {
+          if (status.healthy || (isSDK && status.healthy === undefined)) {
```

Comment:
`status.healthy === undefined` seems like code smell here because I believe `status.healthy` should always be a defined type.